packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -73,11 +72,12 @@ - (void)orientationChanged:(NSNotification *)note {
     return;
   }
 
+  __weak typeof(self) weakSelf = self;
   dispatch_async(self.captureSessionQueue, ^{
     // `FLTCam::setDeviceOrientation` must be called on capture session queue.
-    [self.camera setDeviceOrientation:orientation];
+    [weakSelf.camera setDeviceOrientation:orientation];
     // `CameraPlugin::sendDeviceOrientation` can be called on any queue.
-    [self sendDeviceOrientation:orientation];
```

Comment:
This line is identified as the root cause, but i also went through all usage of "strong self"s